ga4gh/backend.py

```diff
@@ -740,5 +765,4 @@ def __init__(self, dataDir):
             directory != referencesDirName]
         for datasetDir in datasetDirs:
             dataset = datasets.FileSystemDataset(datasetDir)
-            self._datasetIdMap[dataset.getId()] = dataset
-        self._datasetIds = sorted(self._datasetIdMap.keys())
```

Comment:
Are we throwing away the assumption that the datasets are in some kind of order here?  We control the dataset names in the simulated backend, but not in the file system backend.  Should we care?